redisinsight/ui/src/pages/rdi/pipeline-management/pages/jobs/Jobs.tsx

```diff
@@ -18,6 +18,7 @@ import TemplatePopover from 'uiSrc/pages/rdi/pipeline-management/components/temp
 const Jobs = () => {
   const { rdiInstanceId, jobName } = useParams<{ rdiInstanceId: string, jobName: string }>()
   const [decodedJobName, setDecodedJobName] = useState<string>(decodeURIComponent(jobName))
+  const [previousJobName, setPreviousJobName] = useState<string>('')
```

Comment:
Maybe it is better to put previousJobName as ref since we don't need to re-render component